homeassistant/components/template/coordinator.py

```diff
@@ -32,6 +34,18 @@ def __init__(self, hass: HomeAssistant, config: dict[str, Any]) -> None:
         self._unsub_start: Callable[[], None] | None = None
         self._unsub_trigger: Callable[[], None] | None = None
         self._script: Script | None = None
+        self._run_variables: ScriptVariables | None = None
+        self._blueprint_inputs: dict | None = None
+        if config is not None:
+            self._run_variables = config.get(CONF_VARIABLES)
+            self._blueprint_inputs = getattr(config, "raw_blueprint_inputs", None)
+
+    @property
+    def referenced_blueprint(self) -> str | None:
+        """Return referenced blueprint or None."""
+        if self._blueprint_inputs is None:
+            return None
```

Comment:
Can we add a test covering this?

Reply:
I thought this was covered under code coverage, if not, I'll add the test asap